Dockerfile-0.12.6.1

```diff
@@ -0,0 +1,43 @@
+# first stage: build kwkhtmltopdf_server
+
+FROM --platform=linux/amd64 docker.io/golang:1.11
+WORKDIR /tmp
+COPY server/kwkhtmltopdf_server.go .
+RUN go build kwkhtmltopdf_server.go
+
+# second stage: server with wkhtmltopdf
+
+FROM --platform=linux/amd64 docker.io/ubuntu:20.04
+
+RUN set -x \
+  && apt update \
+  && apt -y install --no-install-recommends \
+    wget \
+    ca-certificates \
+    fonts-liberation2 \
+    fontconfig \
+    libjpeg-turbo8 \
+    libx11-6 \
+    libxext6 \
+    libxrender1 \
+    xfonts-75dpi \
+    xfonts-base \
+    fonts-lato \
```

Comment:
Why `fonts-lato`, specifically? Is that commonly used in Odoo reports?

I remember we added fonts-liberation2 because in the past some Odoo reports were sensitive to the specific metric of that font.